### DISCUSSION TITLE
when i use it to mpv3.9，some thing is wrong.

### DISCUSSION THREAD

**Discussion by yanghaoynu** [CogentRedTester/mpv-file-browser#114] [General]
when i use it to mpv 0.39，some thing is wrong.
<img width="636" alt="Snipaste_2024-11-26_15-04-41" src="https://github.com/user-attachments/assets/7d808666-d82f-4fb6-aff5-68309a635351">

**Comment by CogentRedTester** [comment]
What is mpv3.9?

**Reply by yanghaoynu** [reply]
thank you！
sorry，it is mpv player --version 0.39。

**Reply by yanghaoynu** [reply]
@
<img width="1600" alt="Snipaste_2024-11-26_16-58-12" src="https://github.com/user-attachments/assets/1ac801dc-bae7-47e7-af3b-2066cd641a0d">

**Reply by CogentRedTester** [reply]
Are you on Windows?

**Reply by yanghaoynu** [reply]
yes ，OS is windows11 24 h2，I tried both basic and advanced installation methods，Set the parameters  addons=yes in the file_browser.conf ，always i get an error。
1、addons=no：
<img width="960" alt="Snipaste_2024-11-27_08-22-19" src="https://github.com/user-attachments/assets/1ca233c6-0fd2-4478-856f-8d4471381aa1">
2、addons=yes：
<img width="1600" alt="Snipaste_2024-11-27_08-23-26" src="https://github.com/user-attachments/assets/3d37ca5f-1b2f-4ac9-8028-3428751610ec">
3、tab key 
but the The tab key takes effect：
<img width="960" alt="Snipaste_2024-11-27_08-28-55" src="https://github.com/user-attachments/assets/c2a96eb3-9383-4b86-9e79-f2713a69dcc4">
4、mpv log
[mpv.log](https://github.com/user-attachments/files/17927105/mpv.log)
5、file_browser.conf
[file_browser.txt](https://github.com/user-attachments/files/17927117/file_browser.txt)
the end: thanks!

**Reply by yanghaoynu** [reply]
I think ，it is it!
<img width="1565" alt="Snipaste_2024-11-27_09-23-06" src="https://github.com/user-attachments/assets/819caf19-3d98-49f8-ad49-e8c662b49ff5">

**Reply by yanghaoynu** [reply]
but ,need wget for waht?
<img width="960" alt="Snipaste_2024-11-27_09-32-03" src="https://github.com/user-attachments/assets/73a0e1ed-724b-4533-8ccd-9de3d22dd840">

**Reply by CogentRedTester** [reply]
Windir needs the wmic utility, which may be disabled by default on Windows 11. This page seems to show how to enable it: https://techcommunity.microsoft.com/blog/windows-itpro-blog/how-to-install-wmic-feature-on-demand-on-windows-11/4189530.

And yes wget is needed for read-file.

Are you actually using all these addons? Because some of them conflict with each other and others don't even work on Windows.

**Reply by yanghaoynu** [reply]
ok，I got it . only if  set addons=no we can use the  basic features of mpv-file-browser with nothing wrong?if as i say,please tell us how to use.My friends used mpv-file-browser for years ,but never noticed this error!

**Reply by CogentRedTester** [reply]
Just take the addons you're not using out of the `script-modules/file-browser-addons/` directory. If you don't know what the addon does just remove it.
